client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts

```diff
@@ -23,17 +23,36 @@ export function useProcessingLoadingMessages( flow?: string | null ): LoadingMes
 	}
 
 	if ( isWooExpressFlow( flow || null ) ) {
-		return stepData.currentStep === 'siteCreationStep'
-			? [
+		switch ( stepData.currentStep ) {
+			case 'siteCreationStep':
+				return [
 					{
-						title: __( 'Woo! We’re creating your store' ),
+						title: __( "Woo! We're creating your store" ),
```

Comment:
Thanks for the fix @rjchow! 

I think we also need to change the following texts:

https://github.com/Automattic/wp-calypso/blob/46129e3b81e8cc0a7af78757aa4a99dfd143dd28/client/landing/stepper/declarative-flow/internals/steps-repository/assign-trial-plan/index.tsx#L52-L54

https://github.com/Automattic/wp-calypso/blob/46129e3b81e8cc0a7af78757aa4a99dfd143dd28/client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx#L153

Reply:
👀 👀 

I didn't actually realise that I changed the apostrophe! Nor did I expect that the same string is repeated in a few places

Thanks! Will amend this